modules/nf-core/kallisto/quant/main.nf

```diff
@@ -53,11 +57,9 @@ process KALLISTO_QUANT {
             ${strandedness} \\
             ${args} \\
             -o $prefix \\
-            ${reads} 2> >(tee -a ${prefix}.log.txt >&2)
+            ${reads} 2> >(tee -a ${prefix}/kallisto_quant.log >&2)
 
-    if [ -f $prefix/run_info.json ]; then
-        cp $prefix/run_info.json "${prefix}_run_info.json"
-    fi
+    cp ${prefix}/kallisto_quant.log ${prefix}/run_info.json .
```

Comment:
Don't we need to name `${prefix}/kallisto_quant.log` by sample for MultiQC? 🤔 In which case copying this out as it is won't work 😆

Reply:
Actually, to save the hassle later it might be worth making these changes in the kallisto branch to test that the publishing and MultiQC rendering works.